pkg/validating-webhooks/hiveextension/v1beta1/agentclusterinstall_admission_hook.go

```diff
@@ -250,6 +313,13 @@ func areImageSetRefsEqual(imageSetRef1 *hivev1.ClusterImageSetReference, imageSe
 	}
 }
 
+func isUserManagedNetworkingSetToFalseWithSNO(newObject *hiveext.AgentClusterInstall) bool {
+	return newObject.Spec.ProvisionRequirements.ControlPlaneAgents == 1 &&
+		newObject.Spec.ProvisionRequirements.WorkerAgents == 0 &&
+		newObject.Spec.Networking.UserManagedNetworking != nil &&
+		!*newObject.Spec.Networking.UserManagedNetworking
```

Comment:
a question: do you also need to use `swag.bool` for `ewObject.Spec.Networking.UserManagedNetworking` here?

Reply:
This style was actually enforced by the linter. I ca not use swag here because I need to check that the field is not null first